package.json

```diff
@@ -46,7 +46,7 @@
     "heimdalljs-logger": "^0.1.9",
     "json-stable-stringify": "^1.0.1",
     "rsvp": "^4.8.4",
-    "workerpool": "^3.1.1"
+    "workerpool": "^5.0.4"
```

Comment:
what other breaking changes does this include?

Reply:
Version 4.0.0:
[`Bundled library changed due to the upgrade to Webpack 4. This could possibly lead to breaking changes.`](https://github.com/josdejong/workerpool/blob/master/HISTORY.md#2019-08-21-version-400)
Looks like this is the reason for the 4.0.0

Reply:
FWIW, 5.0.4 also officially drops support for Node 8

Reply:
is there any chace this going to be merged soon?
Thanks for the fix